eiffel-vocabulary/EiffelServiceAllocatedEvent.md

```diff
@@ -14,7 +14,7 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 --->
-
+ 
 # EiffelServiceAllocatedEvent  (ServA)
 The EiffelServiceAllocatedEvent is to declare an instance has been allocated to a User.( e.g global scheduler or resource booking tool).
```

Comment:
that infrastructure needed for a Service Instance has 
Ex: A global scheduler or Resource booking tool has allocated the the resources to user